From ddc112358887a05b0ae3951aa749a799d8436f8b Mon Sep 17 00:00:00 2001
From: k2wl <kar2wlan@gmail.com>
Date: Thu, 25 Dec 2014 21:49:50 +0530
Subject: [PATCH] IPCThreadState.cpp

Signed-off-by: k2wl <kar2wlan@gmail.com>
---
 libs/binder/IPCThreadState.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libs/binder/IPCThreadState.cpp b/libs/binder/IPCThreadState.cpp
index d2f784d..9e2606a 100644
--- a/libs/binder/IPCThreadState.cpp
+++ b/libs/binder/IPCThreadState.cpp
@@ -361,6 +361,10 @@ status_t IPCThreadState::clearLastError()
     return err;
 }
 
+extern "C" int _ZN7android14IPCThreadState13getCallingPidEv(IPCThreadState *state) {
+    return state->getCallingPid();
+}
+
 pid_t IPCThreadState::getCallingPid() const
 {
     return mCallingPid;
-- 
2.1.0

